### DISCUSSION TITLE
Is there any bias when using ASReview for screening?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#548] [Q&A] [answered]
When I have multiple themes in my systematic review, would the software then not become biased towards one specific theme?

**Comment by jteijema** [comment, marked as answer]
I've personally worked with datasets containing records regarding six different medical health topics. I've seen that highly clustered data might decrease performance in certain cases, but only temporarily. Once documents from the new cluster were found, the performance increase from having found this new "pool" of clustered records corrected the behaviour. This is not conclusive for other situations of course.

**Reply by SagevdBrand** [reply]
Then it is perhaps a good idea to set relevant papers belonging to one of the topics (as far as one can know all topics within a search) as prior knowledge?

**Reply by jteijema** [reply]
Sounds like a great piece of research :). Get a dataset with multiple topics, and see if having selected priors from each topic has better performance than random priors. Run a big enough sample and there's an answer.

On top of that; run a few simulations with priors from only one specific topic and see if performance drops.
